packages/ketcher-core/src/application/editor/operations/coreRxn/multitailArrow.ts

```diff
@@ -0,0 +1,65 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+/****************************************************************************
+ * Copyright 2021 EPAM Systems
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ***************************************************************************/
+/* eslint-disable @typescript-eslint/no-use-before-define */
+
+import { RenderersManager } from 'application/render/renderers/RenderersManager';
+import { Operation } from 'domain/entities/Operation';
+import { MultitailArrow } from 'domain/entities/CoreMultitailArrow';
+
+export class MultitailArrowAddOperation implements Operation {
+  public multitailArrow: MultitailArrow;
+  public priority = 2;
```

Comment:
I'm thinking maybe we could introduce some priorities mapping so it would be clear which particular operation goes first and it would be easier to maintain in case we'll have some operation which we'll have to perform between current 1 and 2 for example